app/components/api-fetch-container/all-songs.tsx

```
@@ -4,9 +4,10 @@ import { useEffect, useState } from "react"
 import fetchAllSongs from "@/app/api-fetch/all-songs"
 import { CardProps } from "@/app/model/card-props"
 import { processCloudinaryUrl } from "@/app/api-fetch/cloudinary-url-processing"
-import VerticalCard from "@/app/app-components/info-cards/vertical-card"
+import VerticalCard from "@/app/components/info-cards/vertical-card"
 import Skeleton from "../loading/skeleton"
 import ErrorComponent from "./fetch-error"
+import { randomUUID } from "crypto"
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)